src/main/java/org/apache/maven/shared/utils/io/FileUtils.java

```diff
@@ -2103,10 +2117,10 @@ public static boolean isSymbolicLinkForSure( @Nonnull final File file )
     }
 
     /**
-     * @param symlink The link name.
-     * @param target The target.
-     * @return The linked file.
-     * @throws IOException in case of an error.
+     * @param symlink the link name
+     * @param target the target
+     * @return the linked file
+     * @throws IOException in case of an error
      */
     @Nonnull public static File createSymbolicLink( @Nonnull File symlink,  @Nonnull File target )
```

Comment:
What about `java.nio.file.Files.createSymbolicLink(Path, Path, FileAttribute<?>...)`?

Reply:
Perhaps. This method is slightly different, particularly in what happens when the symbolic link already exists. It wraps the `java.nio.file.Files.createSymbolicLink` method for reasons that aren't completely clear to me.

Reply:
OK, I would at least add a `@see` or deprecate and indicate the different behavior.

Reply:
done